.github/workflows/build-container.yml

```diff
@@ -42,12 +42,24 @@ jobs:
         tags=()
         for registry in hub.opensciencegrid.org docker.io; do
           tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO-el$EL_VER )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE )
-          tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
-          if [[ $BASE_YUM_REPO == "release" ]]; then 
-            tags+=( $registry/$docker_repo:latest )
+          if [[ $EL_VER == '8' ]]; then
+            tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
+            tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO )
+            if [[ $BASE_YUM_REPO == 'release' ]]; then
+              tags+=( $registry/$docker_repo:$OSG_RELEASE )
+              if [[ $OSG_RELEASE == '3.6' ]]; then
+                tags+=( $registry/$docker_repo:latest )
+                tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
+              fi
+            else
+              if [[ $OSG_RELEASE == '3.6' ]]; then
+                tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
+              fi
+            fi
```

Comment:
I noticed the duplicate `$OSG_RELEASE == '3.6'` if statements that both added the `$BASE_YUM_REPO` tag, this can be simplified with the following since you're setting the `$BASE_YUM_REPO` for all values of `$BASE_YUM_REPO`:

```suggestion
              [[ $OSG_RELEASE == '3.6' ]] && tags+=( $registry/$docker_repo:latest )  
            fi
            [[ $OSG_RELEASE == '3.6' ]] && tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
```

One nice thing about bash is that there you can chain your statements together, even your comparisons. Here are ways you can chain statements together:

- `&&`: if the first statement returns with exit code 0, run the second statement
- `||`: if the first statement does not return exit code 0, run the second statement
- `;` run the second statement; regardless of the exit status of second statement
- `|`: take the standard output (`stdout`) of the first statement and pass it to the standard in (`stdin`) of the second statement, e.g. `echo 'foo' | grep f`

`man bash` is a very helpful (albeit long) reference, So the above is equivalent to the following:

```
              if [[ $OSG_RELEASE == '3.6' ]]; then
                tags+=( $registry/$docker_repo:latest )
              fi
            if [[ $OSG_RELEASE == '3.6' ]]; then
                tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
            fi
```